Docs/src/tutorials/basictutorials/basictutorial1.md

```diff
@@ -0,0 +1,302 @@
+# Basic Tutorial 1 {#basictutorial1}
+
```

Comment:
the tutorial is rather lenghty - you should add:
`@tableofcontents`
this will require you to move all headers one level up and add a tag i.e. `## How Ogre Works` -> `# How Ogre Works {#howogreworks}`.